arduino/vision_runner/vision_runner.ino

```diff
@@ -15,38 +15,43 @@ double FOV_X = 75; //horizontal Field of View in degrees
 double FOV_Y = 47; //vertical Field of View in degrees
 double RESOLUTION_WIDTH = 320; //in pixels, 320 x 200
```

Comment:
If these are constants, make them `const` https://www.arduino.cc/en/Reference/Const (it's like final)